From 31e9b8d8f301a255a59ee8af1451348da6be8abf Mon Sep 17 00:00:00 2001
From: Rob Johnston <rob.johnston@joyent.com>
Date: Wed, 5 Jun 2019 22:49:18 +0000
Subject: [PATCH] OS-7821 mandoc should allow ERRORS section in all man section
 7 variants Reviewed by: Robert Mustacchi <robert.mustacchi@joyent.com>
 Approved by: Joshua M. Clulow <josh@sysmgr.org>

---
 usr/src/cmd/mandoc/mdoc_validate.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/usr/src/cmd/mandoc/mdoc_validate.c b/usr/src/cmd/mandoc/mdoc_validate.c
index 52545e3c46..32067baf0a 100644
--- a/usr/src/cmd/mandoc/mdoc_validate.c
+++ b/usr/src/cmd/mandoc/mdoc_validate.c
@@ -3,7 +3,7 @@
  * Copyright (c) 2008-2012 Kristaps Dzonsons <kristaps@bsd.lv>
  * Copyright (c) 2010-2018 Ingo Schwarze <schwarze@openbsd.org>
  * Copyright (c) 2010 Joerg Sonnenberger <joerg@netbsd.org>
- * Copyright 2018, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  *
  * Permission to use, copy, modify, and distribute this software for any
  * purpose with or without fee is hereby granted, provided that the above
@@ -2428,9 +2428,9 @@ post_sh_head(POST_ARGS)
 	case SEC_ERRORS:
 		if (*mdoc->meta.msec == '4')
 			break;
-		if (*mdoc->meta.msec == '7' && *(mdoc->meta.msec + 1) == 'I')
+		if (*mdoc->meta.msec == '7')
 			break;
-		goodsec = "2, 3, 4, 7I, 9";
+		goodsec = "2, 3, 4, 7, 9";
 		/* FALLTHROUGH */
 	case SEC_RETURN_VALUES:
 	case SEC_LIBRARY:
-- 
2.21.0

